etna/records/models.py

```diff
@@ -713,6 +720,188 @@ def archive_accessions(self) -> Optional[AccessionsInfo]:
+
+    @cached_property
+    def history(self) -> str:
+        if description := self.get("description", ()):
+            for item in description:
+                if item.get("type") == "history":
+                    if value := item.get("value", ""):
+                        return mark_safe(value)
+        return ""
+
+    @cached_property
+    def biography(self) -> dict:
+        if description := self.get("description", ()):
+            for item in description:
+                if item.get("type") == "biography":
+                    return {"value": item.get("value", ""), "url": item.get("url", "")}
+        return {}
+
+    @cached_property
+    def func_occup_activ(self) -> str:
+        if description := self.get("description", ()):
+            for item in description:
+                if item.get("type") == "functions, occupations and activities":
+                    if value := item.get("value", ""):
+                        document = pq(value)
+                        for tag in ("foa", "function"):
+                            if doc_value := document(tag).text():
+                                return doc_value
+                        return value
+        return ""
+
+    @cached_property
+    def places(self) -> tuple(str):
+        places = ()
+        if place := self.get("place", ()):
+            for item in place:
+                if name := item.get("name", ()):
+                    for item in name:
+                        if value := item.get("value", ""):
+                            places += (value,)
+        return places
+
+    @cached_property
+    def birth_date(self) -> str:
+        return extract(self.get("birth", {}), "date.value", default="")
+
+    @cached_property
+    def death_date(self) -> str:
+        return extract(self.get("death", {}), "date.value", default="")
+
+    @cached_property
+    def start_date(self) -> str:
+        if date := extract(self.get("start", {}), "date", default=()):
+            for item in date:
+                if value := item.get("value", ""):
+                    return value
+        return ""
+
+    @cached_property
+    def end_date(self) -> str:
+        if date := extract(self.get("end", {}), "date", default=()):
+            for item in date:
+                if value := item.get("value", ""):
+                    return value
+        return ""
+
+    @cached_property
+    def record_creators_date(self) -> str:
+        """
+        Returns dates for person, person's activity/service,  both, or any if available
+        """
+        seperator = "-"
+        joiner = "; "
+        person_date = service_activity_date = ""
+        if self.birth_date and self.death_date:
+            person_date = f"{self.birth_date}{seperator}{self.death_date}"
+        else:
+            person_date = self.birth_date or self.death_date
+
+        if self.start_date and self.end_date:
+            service_activity_date = f"{self.start_date}{seperator}{self.end_date}"
```

Comment:
```suggestion
        separator = "-"
        joiner = "; "
        person_date = service_activity_date = ""
        if self.birth_date and self.death_date:
            person_date = f"{self.birth_date}{separator}{self.death_date}"
        else:
            person_date = self.birth_date or self.death_date

        if self.start_date and self.end_date:
            service_activity_date = f"{self.start_date}{separator}{self.end_date}"
```

Just a small spelling change :)

Reply:
Fixed. Thanks :)